### ISSUE TITLE
illegalArgumentException when saving ex. contact

### ISSUE THREAD

**Issue by michaelof** [SimpleMobileTools/Simple-Contacts#634]
6.13.1 F-Droid version
Error pls. see attached screenshot.
Resp. contact also attached.
Error happens reproducible when trying to change type of phone# from cell to work

Regards,
Michael
![Screenshot_20201130-144552](https://user-images.githubusercontent.com/15951622/100618595-57331b00-331c-11eb-8409-384d76d43545.png)

Vcard:
```
BEGIN:VCARD
VERSION:3.0
PRODID:-//Sabre//Sabre VObject 4.3.0//EN
UID:7723e678-322a-4317-bedb-9b3a57f28191
FN:Contabo Support
N:Contabo Support;;;;
ORG:Contabo Support
TEL;TYPE=cell:+498921268372
EMAIL;TYPE=work:support@contabo.de
REV:20201130T134618Z
END:VCARD
```

**Comment by tibbi** [comment]
are you saving it to Phone, SIM, or something else? I mean the last field.

**Comment by michaelof** [comment]
To a cardDAV account. Nexcloud via DAVx5.

**Comment by tibbi** [comment]
should be fixed in 6.14.0
